test/Table_Tests/package.yaml

```diff
@@ -1,6 +1,7 @@
 name: Table_Tests
+namespace: enso_dev
 version: 0.0.1
-enso-version: default
 license: MIT
 author: enso-dev@enso.org
 maintainer: enso-dev@enso.org
+prefer-local-libraries: true
```

Comment:
Another, IMO better, way to do this would be to declare the depended-on library in the edition override:
```
edition:
   extends: 2023.123-dev
   libraries:
     - name: enso_dev.Tests
       repository: local
```

However to do so, we need to specify _some_ edition (like `2023.123-dev` example), but we don't really have one to specify. I guess we should extend the edition resolution to be able to specify `extends: default`. Currently that is not possible, but it should be a relatively simple extension.

@jdunkerley @JaroslavTulach shall I create a ticket to allow `extends: default`? Seems +- aligned with the direction #7639 is going in.

Reply:
> shall I create a ticket .... aligned with the direction https://github.com/orgs/enso-org/discussions/7639 is going in.

#7639 discusses incompatibilities between engine & IDE. I am unsure how that relates and affects editions. If an existing concept like ` prefer-local-libraries` works, it is easier to suggest to use it than think about consequences of changing `extends` mechanism.